### ISSUE TITLE
can't create sessions locally

### ISSUE THREAD

**Issue by aquibbaig** [distributed-system-analysis/pbench-dashboard#83]
When I try to create a session it takes me to a 404 logging the following message:
```
results.data is undefined.
```

**Comment by aquibbaig** [comment]
This was an issue because the graphql endpoint which I was using was not deployed anymore, has been fixed now, hence closing this!

**Comment by aquibbaig** [comment]
Reopening cause cannot add urls in the server

**Comment by aquibbaig** [comment]
@gurbirkalsi Just curious to know about how this works, since `generated` folder doesn't exist inside our src folder this will always throw a `MODULE_NOT_FOUND` in npm start.

```diff
const { ApolloServer } = require('apollo-server');
const typeDefs = require('./typeDefs');
const resolvers = require('./resolvers');

- const { prisma } = require('./generated/prisma-client/index');
+ import prisma from './somewhere';

const server = new ApolloServer({
  typeDefs,
  resolvers,
  context: {
    prisma,
  },
});

server
  .listen({
    port: 8383,
  })
  // eslint-disable-next-line no-console
  .then(info => console.log(`Server started on http://localhost:${info.port}`));
```

**Comment by gurbirkalsi** [comment]
Resolved with PR #89
